crate/cli/src/actions/kms/elliptic_curves/sign.rs

```diff
@@ -0,0 +1,131 @@
+use std::{fs::File, io::Write, path::PathBuf};
+
+use clap::{Parser, ValueEnum};
+use cosmian_kmip::{
+    kmip_0::kmip_types::PaddingMethod,
+    kmip_2_1::{
+        kmip_types::{CryptographicAlgorithm, CryptographicParameters},
+        requests::sign_request,
+    },
+};
+use cosmian_kms_client::{
+    KmsClient, read_bytes_from_file,
+    reexport::cosmian_kms_client_utils::{create_utils::Curve, rsa_utils::HashFn},
+};
+use serde::Deserialize;
+use strum::EnumString;
+
+use crate::{
+    actions::kms::{console, labels::KEY_ID, shared::get_key_uid},
+    error::result::{KmsCliResult, KmsCliResultHelper},
+};
+
+/// Digital signature supported is rsassa-pss
+#[derive(Parser, Debug)]
+#[clap(verbatim_doc_comment)]
+pub struct SignAction {
+    /// The elliptic curve
+    #[clap(long = "curve", short = 'c', default_value = "nist-p256")]
+    pub(crate) curve: Curve,
+
+    /// The file to decrypt
+    #[clap(required = true, name = "FILE")]
+    pub(crate) input_file: PathBuf,
+
+    /// The private key unique identifier
+    /// If not specified, tags should be specified
+    #[clap(long = KEY_ID, short = 'k', group = "key-tags")]
+    pub(crate) key_id: Option<String>,
+
+    /// Tag to use to retrieve the key when no key id is specified.
+    /// To specify multiple tags, use the option multiple times.
+    #[clap(long = "tag", short = 't', value_name = "TAG", group = "key-tags")]
+    pub(crate) tags: Option<Vec<String>>,
+
+    /// The signature algorithm
+    #[clap(
+        long = "signature-algorithm",
```

Comment:
```suggestion
        long,
```

clap will automatically set the "long" param to "signature-algorithm"